pulpcore/app/access_policy.py

```diff
@@ -23,13 +23,28 @@ def get_policy_statements(self, request, view):
 
             AccessPolicyModel.objects.get(viewset_name=get_view_urlpattern(view))
 
+        If a matching `pulpcore.plugin.models.AccessPolicy` cannot be found, a default behavior of
+        allowing any admin user to perform any operation is used. This fallback allows the Pulp RBAC
+        implementation to be turned on endpoint-by-endpoint with less effort. Once RBAC is fully
+        implemented on all endpoints, this fallback will be removed.
+
         Args:
             request (rest_framework.request.Request): The request being checked for authorization.
             view (subclass rest_framework.viewsets.GenericViewSet): The view name being requested.
 
         Returns:
             The access policy statements in drf-access-policy policy structure.
         """
-        access_policy_obj = AccessPolicyModel.objects.get(viewset_name=get_view_urlpattern(view))
-
-        return access_policy_obj.statements
+        allow_authenticated = [{"action": "*", "principal": "admin", "effect": "allow"}]
+
+        try:
+            viewset_name = get_view_urlpattern(view)
+        except AttributeError:
+            return allow_authenticated
+
+        try:
+            access_policy_obj = AccessPolicyModel.objects.get(viewset_name=viewset_name)
+        except AccessPolicyModel.DoesNotExist:
+            return allow_authenticated
+        else:
+            return access_policy_obj.statements
```

Comment:
```suggestion
        try:
            access_policy_obj = AccessPolicyModel.objects.get(viewset_name=get_view_urlpattern(view))
        except AttributeError, AccessPolicyModel.DoesNotExist:
            return allow_authenticated
        return access_policy_obj.statements
```

Also maybe rename `allow_authenticated` to `allow_isadmin`.

Reply:
I originally had this but thought maybe it would be better the other way. This is so nicely compact though, I'll switch back to it.